tasks/sanity-inspect-image.yaml

```diff
@@ -0,0 +1,24 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: sanity-inspect-image
+spec:
+  params:
+    - name: sourceImage
+      description: the fully qualified image name
+      default: quay.io/hongweiliu/skopeo-inspect-test:latest
+      type: string
+  steps:
+  - name: inspect-image
+    image: quay.io/redhat-appstudio/hacbs-test:latest
+    script: |
+      if [ -f "$(workspaces.conftest-ws.path)/image_inspect.json" ];then
```

Comment:
Same comment as the deprecated image check around the design patterns. I know it's perfectly legal to embed scripting in a tekton task but I'm leaning towards us putting a lot of this logic in scripts/tools in the hacbs-test repo. So in this case encapsulate all this skopeo logic from both tasks into a lightweight script in our hacbs-test repo. That way we can make the changes against the script rather than against the task. Keeps the tasks cleaner and less change since it's always interfacing with the same entry point. Thoughts?

Reply:
Thanks. I understand. I checked the script, should we keep consistent with the tasks in other workstream? Or we may get a agreement for it.

Reply:
I think EC has wrapped some of their stuff in wrapper scripts that their task interfaces with https://github.com/redhat-appstudio/build-definitions/blob/main/tasks/cosign-verify-attestation.yaml#L30 which is a script https://github.com/redhat-appstudio/build-definitions/blob/main/appstudio-utils/util-scripts/cosign-verify-attestation.sh

But we don't have to do it as part of this PR. We can just merge it and put a ticket in the backlog to refactor some of these things out into scripts in our hacbs-test repo.

Reply:
@Dannyb48 Thank you!